### ISSUE TITLE
Error on registry login

### ISSUE THREAD

**Issue by allurefx** [JAFLabs/wa.dev-issues#3]
I just signed up, but see the below error on registry login.
```
error: failed to get signing: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files
```

**Comment by calvinrp** [comment]
`sudo apt-get install -y gnome-keyring` should fix this. Let me know.

The Warg CLI relies on OS functionality for securely managing on-device secrets. Linux installs appear to lack the keyring, by default. When we publish the Warg CLI to the OS package managers, this dependency issue should be simplified as part of the install.

Sorry for the delay in seeing your message, trying to adjust my GitHub notifications.

**Comment by calvinrp** [comment]
Just released a fix. Update the `warg` client to `v0.4.2` or higher. Fixed with https://github.com/bytecodealliance/registry/pull/270

`cargo install warg-cli`

**Comment by calvinrp** [comment]
Actually, not quite solved yet. See https://github.com/bytecodealliance/registry/issues/271

**Comment by calvinrp** [comment]
Fixed in `warg-cli` release `0.7.0`.

There is better error messaging and suggestions when keyring service is not available. 
Also, you can configure a different keyring backend, such as `warg config --keyring-backend flat-file`.
